docs/reference/index-modules/translog.asciidoc

```diff
@@ -59,8 +60,9 @@ update, or bulk request.  This setting accepts the following parameters:
 `async`::
 
     `fsync` and commit in the background every `sync_interval`. In
-    the event of hardware failure, all acknowledged writes since the last
-    automatic commit will be discarded.
+    the event of hardware failure or operating system crash or JVM crash
```

Comment:
Given that this is repeating what's been written above, perhaps just write here "In the event of a failure, all acknowledged writes since the last automatic commit will be discarded".